src/client/src/AppContext.ts

```diff
@@ -12,7 +12,7 @@ interface IAppContext {
 //TODO: USE LAUNCH COMMAND HERE. When you choose that from Command Palette:
 //TODO: Web Template Studio: Create Web App
 //TODO: Web Template Studio: Create React Native App
-const devPlatform = PLATFORM.WEB;
-export const platform = (process.env.NODE_ENV === ENVIRONMENT.DEVELOPMENT) ? devPlatform : PLATFORM.WEB;
+const devPlatform = PLATFORM.REACTNATIVE;
```

Comment:
if we have the command, see if we can just rely on that somehow befor completing. If not jus leave WEB